### ISSUE TITLE
Image preview not working as expected on iOS

### ISSUE THREAD

**Issue by mykdavies** [liftoff-app/liftoff#336]
### Device info

- OS: iOS

### Describe the bug

User report: "Tapping a photo brings up the expanded view of the photo which is good. The options bar at the top of the screen partially obscures the photo, and normally in iOS tapping the photo again hides the options bar, but just closes the photo in this app without a clear alternative to get rid of the options bar. Possibly a swipe left should close the photo while a tap hides/reveals the options bar"

### Steps to reproduce

1. Tap on any image in a post.
2. Tap again to remove overlay bar.
3. Image closes instead.

### Expected behavior

The overlay should disappear until restored by another tap.
Maybe use swipe to go back to previous screen -- fits with our standard gestures?

### Additional context

"[Cheers, as an apollo refugee, you’re doing god’s work]"

**Comment by jjcomer** [comment]
This was added as a feature (tap to close the image). Let me take a look to see if there is a better UX we could use

**Comment by mykdavies** [comment]
Yeah, I think the image viewer widget is nicely designed but that means it finds uses for the main gestures. I don't think swipe right was taken as it used something else for dismiss, swipe up maybe?
